crypto/rand/drbg_lib.c

```diff
@@ -78,29 +78,29 @@ typedef struct drbg_nonce_global_st {
 static const char ossl_pers_string[] = DRBG_DEFAULT_PERS_STRING;
 
 #define RAND_DRBG_TYPE_FLAGS    ( \
-    RAND_DRBG_FLAG_MASTER | RAND_DRBG_FLAG_PUBLIC | RAND_DRBG_FLAG_PRIVATE )
+    RAND_DRBG_FLAG_PARENT | RAND_DRBG_FLAG_PUBLIC | RAND_DRBG_FLAG_PRIVATE )
 
-#define RAND_DRBG_TYPE_MASTER                     0
+#define RAND_DRBG_TYPE_PARENT                     0
 #define RAND_DRBG_TYPE_PUBLIC                     1
 #define RAND_DRBG_TYPE_PRIVATE                    2
 
 /* Defaults */
 static int rand_drbg_type[3] = {
-    RAND_DRBG_TYPE, /* Master */
+    RAND_DRBG_TYPE, /* Parent */
     RAND_DRBG_TYPE, /* Public */
     RAND_DRBG_TYPE  /* Private */
 };
 static unsigned int rand_drbg_flags[3] = {
-    RAND_DRBG_FLAGS | RAND_DRBG_FLAG_MASTER, /* Master */
+    RAND_DRBG_FLAGS | RAND_DRBG_FLAG_PARENT, /* Main */
     RAND_DRBG_FLAGS | RAND_DRBG_FLAG_PUBLIC, /* Public */
     RAND_DRBG_FLAGS | RAND_DRBG_FLAG_PRIVATE /* Private */
 };
 
-static unsigned int master_reseed_interval = MASTER_RESEED_INTERVAL;
-static unsigned int slave_reseed_interval  = SLAVE_RESEED_INTERVAL;
+static unsigned int parent_reseed_interval = PARENT_RESEED_INTERVAL;
```

Comment:
I'm an orphan and parent sounds offensive for me. Could we change this to something else?

Reply:
@pradyunsg Ok, here's a quote from @richsalz

> While it's okay that you don't find the terms objectionable, many do, and you should similarly be okay with that as well, and respectful of their feelings no matter how nonsensical you feel it is.

https://github.com/openssl/openssl/pull/12089#issuecomment-642317580

Reply:
I don't think opinions expressed by an anonymous account should be considered when making a decision on this topic.

---

@blacklistisnotracist is an anonymous account that:

- was made 2 days ago, which is *after* a global, multi-country social justice movement has gained significant momentum
- has a clear (political) agenda that is meant to push (see username)
- has [a profile picture belittling the entire premise of said movement](https://web.archive.org/web/20200611152529/https://avatars3.githubusercontent.com/u/66671159)
- is making claims that cannot be verified, or attributed to a person

All these signals point to this account being used *only* to disrupt the conversation taking place here (and similar conversations elsewhere) and is being operated as an online troll. I see no reason to believe that this account is operated by someone in good faith.

Reply:
@pradyunsg

Nice try. You really sound like you're someone who judge others with their labels.

I'm in good faith. Labeling innocent words as problematic or having racist connotations, is simply detrimental. Blacklist is a very good instance here. Nobody really have thought blacklist carries any racist connotations. But, because someone (and most of them are white people who imagined blacklist could be sensitive out of their white guilt) started to claiming so, others who never thought that way now have to care about it. That is nothing but ridiculous. I don't like to see people creating a completely new discriminative word out of nowhere.

Reply:
I'm not looking at labels, but [actions](https://github.com/blacklistisnotracist/the-most-politically-correct-software/commit/77b28f0926478eae1f2d9bc52cb180b288d3ad6b). None the less, I'm not interested in further discussion about the nature of your account and your intents here. Please don't @-mention me further.